jax-inference-offloading/jax_inference_offloading/vllm/extension.py

```diff
@@ -51,14 +52,24 @@ def device_info(self):
     )
 
   def set_sharding(self):
-    for _, module in self.model_runner.model.named_modules():
+
+    try:
```

Comment:
Please add some documentation to explain what's going on here and when to remove it.

Reply:
Let me know how the newly-added comments look: https://github.com/NVIDIA/JAX-Toolbox/blob/5591ce449ec20da26b339ae83609f3ec16cb6c45/jax-inference-offloading/jax_inference_offloading/vllm/extension.py#L55-L75

Reply:
Is there already a bug or discussion we can reference?